tg_base/wizard/base_partner_merge.py

```diff
@@ -0,0 +1,10 @@
+from odoo import models
+
+
+class MergePartnerAutomatic(models.TransientModel):
+    _inherit = "base.partner.merge.automatic.wizard"
+
+    def _merge(self, partner_ids, dst_partner=None, extra_checks=True):
+        if self.env.user.has_group("account.group_account_user"):
+            extra_checks = False
+        return super()._merge(partner_ids, dst_partner, extra_checks)
```

Comment:
Изначально в этот модуль в 14.0 вводилась возможность бухгалтерам с правами account.group_account_user сливать партнеров с разными записями account.move и account.move.line, поскольку основной метод `_merge` не позволял это делать.

Но после коммита https://github.com/odoo/odoo/commit/0148f19f67f8725ae9bea7c208f09b4676652cc0 оригинальный _merge уже позволяет это делать.

Соотвественно предлагаю метод удалить целиком и вместо него оставить это:
```
# removed since
# https://github.com/odoo/odoo/commit/0148f19f67f8725ae9bea7c208f09b4676652cc0
```